nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/scheduling/StandardProcessScheduler.java

```diff
@@ -138,7 +138,6 @@ public void setMaxThreadCount(final SchedulingStrategy schedulingStrategy, final
             return;
         }
 
-        agent.setMaxThreadCount(maxThreadCount);
```

Comment:
Was this intended? It appears this method `setMaxThreadCount` no longer does anything. Assuming it was intended, we can likely remove the rest of the code in this method. If the method is no longer relevant, should it be deprecated?